containers/kubernetes/reference-content/version-support-policy.mdx

```diff
@@ -89,4 +102,34 @@ The following documentation lists only the main feature changes for each version
 * Exec Probe Timeout Handling
 
 ### 1.19
-Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
\ No newline at end of file
+Official [release announcement](https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/)
+
+## Supported Container Runtime Interfaces (CRI)
```

Comment:
Similar to this section, we do need another ## section below this one, named Supported Container Network Interfaces (CNI)